web/ui/react-app/src/App.css

```diff
@@ -135,16 +135,42 @@ div.time-input {
 }
 
 .graph-legend {
-    margin: 15px 0 15px 25px;
-    font-size: 0.8em;
+    margin: 15px 0 15px 55px;
+    font-size: 0.75em;
+    background: #222;
+    color: white;
+    padding: 10px 5px;
+    display: inline-block;
+    cursor: pointer;
```

Comment:
Hmm, I don't think we should be trying to copy the classic UI's legend styling here, just getting the series selection behavior to work. The old legend always looked a bit out of place in the non-dark page design. I think the only reason it looked like it did is because originally I copied it from a Rickshaw graphing library example like https://tech.shutterstock.com/rickshaw/examples/legend.html.

Of course we have to add some hover styles etc., but maybe just adjust them for the light design?

Reply:
Ok, let's style it in another PR.

Reply:
I don't think we should be introducing a completely new styling in this PR though just to roll it back again. Or do you mean you were still planning on removing the new dark style in this PR?

Reply:
Ok, I'll revert the old styles. Are there any plans to be styled differently?

Reply:
No, I mean we only need new styles for the new functionality (hover highlight, selected series, etc.), but not change the whole design otherwise.